tests/snmp/test_snmp_interfaces.py

```diff
@@ -120,9 +120,9 @@ def verify_snmp_speed(facts, snmp_facts, results):
     return results
 
 @pytest.mark.bsl
-def test_snmp_interfaces(localhost, creds, duthosts, enum_dut_hostname, enum_asic_index):
+def test_snmp_interfaces(localhost, creds, duthosts, enum_rand_one_per_hwsku_hostname, enum_asic_index):
```

Comment:
This will check interfaces on supervisor node too, right? Was the original intention to use fixture `enum_rand_one_per_hwsku_frontend_hostname`?

Reply:
@wangxin the reason on doing  `enum_rand_one_per_hwsku_hostname` is because the other test test_snmp_mgmt_interface is applicable to both supervisor and front end node and we found if you have both  `enum_rand_one_per_hwsku_hostname` and  `enum_rand_one_per_hwsku_frontend_hostname` in same module correct nodes are not selected, I will add and if condition to skip this test for supervisor for now